server/config/auth.js

```diff
@@ -26,11 +28,11 @@ passport.deserializeUser((user, done) => {
 });
 
 const authenticate = passport.authenticate('google', {
-  scope: ['https://www.googleapis.com/auth/plus.login']
+  scope: ['https://www.googleapis.com/auth/plus.login', 'email', 'profile']
 });
 
 const authenticateCallback = passport.authenticate('google', {
-  failureRedirect: '/'
+  failureRedirect: FAILURE_URL
```

Comment:
why do we have to redirect to failure_url ??

Reply:
this code was removed